src/diffusers/utils/loading_utils.py

```diff
@@ -47,3 +50,71 @@ def load_image(
         image = image.convert("RGB")
 
     return image
+
+
+def load_video(
```

Comment:
This will automatically be removed after my animatediff controlnet PR is merged :)